dockstore-webservice/src/main/java/io/dockstore/webservice/resources/AliasableResourceInterface.java

```diff
@@ -49,27 +56,75 @@
      */
     T getAndCheckResourceByAlias(String alias);
 
-    default T addAliases(User user, Long id, String aliases) {
-        T c = getAndCheckResource(user, id);
-        Set<String> oldAliases = c.getAliases().keySet();
-        Set<String> newAliases = Sets.newHashSet(Arrays.stream(aliases.split(",")).map(String::trim).toArray(String[]::new));
+    /**
+     * Check that aliases do not contain invalid prefixes
+     * if the user adding them is not an admin or curator
+     * @param aliases a Set of alias strings
+     * @param user user authenticated to issue a DOI for the workflow
+     * @param blockAliasesWithParticualarFormat block creation of an alias with a particular format
```

Comment:
Should be `blockAliasesWithParticularFormat` to fix the typo, but more descriptively should be `blockZenodoAliases` or similar

Reply:
fixed